src/character.h

```diff
@@ -1284,6 +1284,10 @@ class Character : public Creature, public visitable
          *  Uses has_opposite_trait(), has_lower_trait(), and has_higher_trait() to determine conflicts.
          */
         bool has_conflicting_trait( const trait_id &flag ) const;
+        /**
+         * Returns true if the player has a threshold mutation
+         */
+        bool has_threshold() const;
```

Comment:
Is it a problem that has_threshold already exists in character.h and is referring to the aiming code?  

```
 std::string action;
  366      std::string help;
  367:     bool has_threshold;
  368      int threshold;
  369      bool operator==( const aim_type &other ) const {
  370:         return has_threshold == other.has_threshold &&
  371                 threshold == other.threshold &&
  372                 name == other.name &&
```

Reply:
Shouldn't be, that's just a variable on a different object type